include/deal.II/hp/fe_collection.h

```diff
@@ -289,6 +289,58 @@ namespace hp
      * numbers::invalid_unsigned_int.
      */
     unsigned int
+    find_least_face_dominating_fe_in_collection(
+      const std::set<unsigned int> &fes) const;
+
+    /**
+     * Try to find a least dominant finite element inside this FECollection
```

Comment:
You can use ‘@copydoc function_name’ to avoid this.

Just keep the note that it’s deprecate

Reply:
Uh nice, I didn't know about that. I'll fix it.